package.json

```diff
@@ -16,42 +16,42 @@
     "posttest": "npm audit",
     "test:lint": "eslint .",
     "test:unit": "nyc _mocha",
-    "prepare": "husky install"
+    "prepare": "husky"
   },
   "dependencies": {
-    "async": "^3.2.5",
-    "body-parser": "^1.20.2",
+    "async": "^3.2.6",
+    "body-parser": "^2.0.1",
```

Comment:
This looks like an unstable version, `latest` is still `1.20.3` so we can stick with that for now. We can have another look at this when we look into updating to Express 5.